README.md

```diff
@@ -133,6 +149,30 @@ nagios-nrpe
 %end
 ```
 
+You can pass a list of packages, with options, to install using the `packages` parameter:
+
+```
+kickstart { '/var/www/html/kickstart.cfg':
+  packages => {
+    'options' => '--nobase --ignoremissing',
+    'package_list' => [
```

Comment:
**nit pick alert!** can you align the arrows for the `options` and `package_list` keys?